### DISCUSSION TITLE
Can anyone tell me what is wrong with my website?

### DISCUSSION THREAD

**Discussion by dhruvkrishnavaid** [community/community#22721] [Other Feature Feedback, Questions, & Ideas]
<p>Can anyone tell me what’s wrong with my github pages subdomain???<br>
I created this website totally from scratch, and it was working fine locally. But when I uploaded it to Github servers, the “follow me on github” button is not working as expected. Why???<br>
It is working locally, but not online… Why???<br>
Is this a problem with my code??? If yes, then please highlight the code snippet that seems wrong. Please.<br>
I am a newbie to github.<br>
My Github username: <strong>dhruvkrishnavaid</strong><br>
My Github Pages subdomain: <strong><a href="http://dhruvkrishnavaid.github.io" rel="nofollow noopener">dhruvkrishnavaid.github.io</a></strong><br>
My Repository: <strong><a href="https://github.com/dhruvkrishnavaid/dhruvkrishnavaid.github.io/" rel="nofollow noopener">https://github.com/dhruvkrishnavaid/dhruvkrishnavaid.github.io/</a></strong></p>

**Comment by SethClydesdale** [comment]
<p>Hello <a class="mention" href="/dhruvkrishnavaid">@dhruvkrishnavaid</a>, welcome to the GitHub Support Community!</p>
<p>Just to clarify: what is the expected behavior of the “Follow me on GitHub” button? When checking your website, the button appears to be normal and it also leads to <a href="https://github.com/dhruvkrishnavaid/" rel="nofollow noopener">your profile on GitHub</a> when clicked.</p>

**Comment by dhruvkrishnavaid** [comment]
<p><a class="mention" href="/sethclydesdale">@sethclydesdale</a> Yes, It’s working fine but with a small glitch.<br>
The expected behaviour, about which I mentioned, was that the link should be spread all over the button. In Google Chrome, it was just upto it’s upper edge.<br>
It maybe a browser or viewport issue. Because when I opened it on Microsoft Edge, it was working fine. Earlier I used Google Chrome. :astonished::astonished:</p>
<p>Note the location at which the colour of the button changed in both the cases.</p>
<p>Issue with google Chrome: <a href="https://gofile.io/d/4MQgvz" rel="nofollow noopener">https://gofile.io/d/4MQgvz</a><br>
Resolved in Microsoft Edge: <a href="https://gofile.io/d/rxNnlJ" rel="nofollow noopener">https://gofile.io/d/rxNnlJ</a></p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Ah, you’re correct! Thanks for clarifying. :+1:</p>
<p>It is indeed an issue with viewports. Adjusting my browser window or opening the devtools docked to the bottom displaces the <code>.home-feature</code> section, causing it to overlap the button and block it from being clicked.</p>
<p>The negative top margin on the <code>.home-feature</code> class appears to be causing the overlap.</p>
<pre><code class="lang-css">.home-feature {
  padding-bottom: 0px;
  padding-top: 30px;
  margin-top: -220px;
  z-index: 9;
}
</code></pre>
<p>So getting rid of that would fix it. However, if it’s a necessary part of your design, you may need to adjust the <code>margin-top</code> value with a media query, like the one below.</p>
<pre><code class="lang-css">@media (min-width:991px) and (max-height:680px) {
  .home-feature { margin-top:-150px; }
}
</code></pre>
<p>Using the device mode in Chrome, I could deduce that it happens when the viewport’s height is around <code>680px</code>.</p>

**Comment by dhruvkrishnavaid** [comment]
<p>Thanks for your help… :kissing_heart::kissing_heart::kissing_heart:<br>
The media query worked. It was indeed a viewport conflict.<br>
Thank you sooo much. :heart::heart::heart:</p>

**Comment by SethClydesdale** [comment]
<p>You’re welcome! Glad to hear that fixed the problem. :slight_smile:</p>
<p>Have a good day.</p>
